workshops/templates/workshops/all_airports.html

```diff
@@ -3,11 +3,6 @@
 {% load crispy_forms_tags %}
 {% load pagination %}
 
-{% block sidebar %}
-    <h3>Filter</h3>
-    {% crispy filter.form form_helper %}
-{% endblock %}
```

Comment:
Okay, I can see why (because the sidebar's form is included one level higher in templates hierarchy https://github.com/swcarpentry/amy/pull/871/files#diff-ee7eb5718e08d7967a72f5c6dbc4860bR12 )